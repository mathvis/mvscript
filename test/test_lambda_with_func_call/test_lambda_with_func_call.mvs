func b() {}
((a: int): {})(b())
