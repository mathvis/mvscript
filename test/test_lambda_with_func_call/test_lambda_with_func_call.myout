Decl (FunctionDef (FunctionIdentifier "b") [] IntT Nothing)
Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",IntT)] (Just (Block (FunctionBlock VoidT) []))) (FunctionCall (FunctionIdentifier "b") []))
