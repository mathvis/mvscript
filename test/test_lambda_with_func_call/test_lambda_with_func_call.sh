#!/bin/bash
helix test_lambda_with_func_call.mvs test_lambda_with_func_call.out --vsplit
