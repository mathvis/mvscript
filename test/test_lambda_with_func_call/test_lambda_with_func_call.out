Decl (FunctionDef (FunctionIdentifier "b") [] VoidT (Just (Block (FunctionBlock VoidT) [])))
Expr (LambdaApplication (LambdaFunc [(VarIdentifier "a",IntT)] (Just (Block (FunctionBlock VoidT) []))) (FunctionCall (FunctionIdentifier "b") []))
